gateway/gateway.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright the Hyperledger Fabric contributors. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+option go_package = "github.com/hyperledger/fabric-protos-go/gateway";
+option java_multiple_files = true;
+option java_package = "org.hyperledger.fabric.protos.gateway";
+option java_outer_classname = "GatewayProto";
+
+package protos;
+
+import "peer/proposal.proto";
+import "common/common.proto";
+
+service Gateway {
+    rpc Endorse(ProposedTransaction) returns (PreparedTransaction) {}
+    rpc Submit(PreparedTransaction) returns (stream Event) {}
+    rpc Evaluate(ProposedTransaction) returns (Result) {}
+}
+
+// The value that is returned by the transaction function
+message Result {
+    bytes value = 1;
```

Comment:
Fabric has tons of these opaque "bytes" fields in messages that really contain some serialized protocol buffer message. As we document these fields, can we please indicate what they typically contain?